### DISCUSSION TITLE
Automation test of Angular app hosted under CefSharp browser in WPF by using Cypress

### DISCUSSION THREAD

**Discussion by phaterster** [cypress-io/cypress#22618] [Questions and Help]
Hi,
We have a WPF application that uses cefsharp browser to host web site (Angular) - in other words - hybrid application.

We're interested to run an automation test in Cypress. Before we had implemented testing this using protractor and it was working well by adding in the protractor configurator: 'chromeOptions': { 'debuggerAddress': "127.0.0.1:8088" } and setting the appropriate baseUrl. 

My question is it possible to test this in a similar way to protractor but using Cypress?

**Comment by zliant** [comment]
@phaterster , any update of this question?  my cefsharp also not work with cypress now.
